Keymaps/Default (Linux).sublime-keymap

```diff
@@ -152,6 +152,7 @@
     // Go To Definition
     // {
     //     "command": "lsp_symbol_definition",
+    //     // "args": { "side_by_side": true },
```

Comment:
Could also uncomment `args` and set to `false` by default. I guess less friction that way?

Reply:
Yes, just uncomment `args`. Also, I formatted all the json with vscode-json-language-server, perhaps we can keep the formatting consistent?